src/Components/NewClient.js

```diff
@@ -1,32 +1,101 @@
-import React from "react";
+import React, { useState } from "react";
 import "./NewClient.css";
 
-class NewClient extends React.Component {
-  render() {
-    return (
-      <section className="new-client">
+export default function NewClient() {
+  // const [ firstName, setFirstName ] = useState("");
+  // const [ lastName, setLastName ] = useState("");
+  // const [ phone, setPhone ] = useState("");
+  // const [ email, setEmail ] = useState("");
+  
+  // const handleUserInput = () => {
+  //   const formInputs = {
+  //     "firstName": firstName,
+  //     "lastName": lastName,
+  //     "phone": phone,
+  //     "email": email,
+  //   }
+  //   console.log(formInputs)
+  // }
+
+  const [formInput, setFormInput] = useState({})
+
+  const handleUserInput = (e) => {
+    setFormInput({...formInput, [`${e.target.id}`]: e.target.value});
+  }
+
+  return (
+    <section className="new-client">
         <h4>New Client Info</h4>
         <div>
           <form>
             <label htmlFor="firstName">First Name</label>
-            <input type="text" id="firstName" />
+            <input type="text" id="firstName" value={formInput.firstName ? formInput.firstName : ""} onInput={e => handleUserInput(e)} />
             <label htmlFor="lastName">Last Name</label>
-            <input type="text" id="lastName" />
+            <input type="text" id="lastName" value={formInput.lastName ? formInput.lastName : ""} onInput={e => handleUserInput(e)} />
             <label htmlFor="phone">Telephone</label>
-            <input type="tel" id="phone" />
+            <input type="tel" id="phone" maxLength="15" placeholder="0" value={formInput.phone ? formInput.phone : ""} onInput={e => handleUserInput(e)} />
```

Comment:
```suggestion
            <input type="tel" id="phone" maxLength="15" placeholder="(xxx) xxx-xxxx" value={formInput.phone ? formInput.phone : ""} onInput={e => handleUserInput(e)} />
```
It's up to you if you want to have placeholders for your inputs, but I think "0" is not the best placeholder for a phone number. I've seen a lot of forms online use something like "(xxx) xxx-xxxx" or "(999) 999-9999". 

In reality, getting a phone input right when it comes to UX design is a little more tricky than just using a placeholder value, which is out of the scope of this lab, but here's an article about it in case you're interested: https://uxmovement.com/forms/bad-practices-on-phone-number-form-fields/